client/src/components/SearchBox.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+export default function SearchBox() {
+    return (
+        <div className="d-flex justify-content-center align-items-center mt-3 mx-auto" id="search-box">
+            <input type='search' className='form-input p-3 is-valid' placeholder="Search For..." />
```

Comment:
Fix inconsistent use of single and double quotes for attribute values.

Reply:
solved